relay/relay_OCD_dB_Manager.ash

```diff
@@ -418,28 +418,43 @@ void add_items() {
 
 	int AddQ;
 	foreach key in OCDefault
-		if(!(OCD contains key) && item_amount(key) > 0) AddQ += 1; #{AddQ += 1; print(key);}
+		if(!(OCD contains key) && (item_amount(key) > 0 || equipped_amount(key) > 0 || closet_amount(key) > 0 || storage_amount(key) > 0 || display_amount(key) > 0)) AddQ += 1; #{AddQ += 1; print(key);}
 	int curr_items = curr_items();
 	if(curr_items > 0 && AddQ > 0) {
 		page.append("<p>");
 		if(write_button("defaultdata", "Add default")) {
 			foreach key in OCDefault
-				if(!(OCD contains key) && item_amount(key) > 0) OCD[key] = OCDefault[key];
+				if(!(OCD contains key) && (item_amount(key) > 0 || equipped_amount(key) > 0 || closet_amount(key) > 0 || storage_amount(key) > 0 || display_amount(key) > 0)) OCD[key] = OCDefault[key];
 			save_ocd();
 			curr_items = curr_items();
 		}
 		page.append(" Add default information for "+AddQ+" common item"+(AddQ == 1? " that is": "s that are")+" listed below.</p>");
 	}
 
 	boolean table_started = false;
-	foreach doodad in get_inventory()
+	int[item] doodads;
+	foreach doodad in $items[]
+	{
+		if (item_amount(doodad) > 0)
+			doodads[doodad] += item_amount(doodad);
+		if (equipped_amount(doodad) > 0)
+			doodads[doodad] += equipped_amount(doodad);
+		if (closet_amount(doodad) > 0)
+			doodads[doodad] += closet_amount(doodad);
+		if (storage_amount(doodad) > 0)
+			doodads[doodad] += storage_amount(doodad);
+		if (display_amount(doodad) > 0)
+			doodads[doodad] += display_amount(doodad);
+	}
+
+	foreach doodad in doodads
 		// Quest items are the only items that cannot be displayed, so check for is_OCDable()
 		if(is_OCDable(doodad) && !(OCD contains doodad && OCD[doodad].action != "UNKN")) {
 			if(!table_started) {
 				page.add_catbuttons();
 				
 				page.append("<table border=0 cellpadding=1>");
-				page.append("<tr><th colspan=2>Item</th><th>Have</th>");
+				page.append("<tr><th colspan=2>Item</th><th>Have</th><th>Closet</th><th>Storage</th><th>Display</th>");

```

Comment:
```suggestion
				page.append("<tr>");
				page.append("<th colspan=2>Item</th>");
				page.append("<th>Have</th>");
				page.append('<th><abbr title="Closet">C</abbr></th>');
				page.append('<th><abbr title="Storage">S</abbr></th>');
				page.append('<th><abbr title="Display Case">DC</abbr></th>');
```

The header row is getting crowded. We could use abbreviations to make it a bit compact.